### DISCUSSION TITLE
[React] lifecycle

### DISCUSSION THREAD

**Discussion by hochan222** [transcendence42/javascript-archive#33] [React]
주제 키워드는

✨lifecycle✨

입니다.

> 모든 라이프 사이클 메소드를 한번씩 사용해 보아요. React Developer Tools를 사용하면서 변화를 관찰도 같이 해보아요!

#### ☘️  관련 주제 키워드

- hooks (useEffect)
- reconciliation algorithm

#### ⏳ 마감 기한

6월 12일 토요일 

⏳스레드에 각 페어당 소요시간 적어주세요!

(ex) holee 90분 소요

**Comment by yechoi42** [comment]
## 라이프 사이클

![image](https://user-images.githubusercontent.com/61399588/121874087-ff04b900-cd41-11eb-94d1-36d71f25d8f7.png)

https://www.zerocho.com/category/React/post/579b5ec26958781500ed9955
